selfdrive/car/hyundai/values.py

```diff
@@ -1734,6 +1736,18 @@ class Buttons:
       b'\xf1\x00SG2_ RDR -----      1.00 1.01 99110-AT000         ',
     ],
   },
+  CAR.GENESIS_GV80: {
+    (Ecu.fwdCamera, 0x7c4, None): [
+      b'\xf1\x00JX1 MFC  AT USA LHD 1.00 1.02 99211-T6110 220513',
+    ],
+    (Ecu.fwdRadar, 0x7d0, None): [
+      b'\xf1\x8799110T6100\xf1\x00JX1_ SCC FHCUP      1.00 1.01 99110-T6100         ',
+      b'\xf1\x00JX1_ SCC FHCUP      1.00 1.01 99110-T6100         ',
+    ],
+    (Ecu.cornerRadar, 0x7b7, None): [
+      b'\xf1\x00JX1 BCW RR 1.00 , 1.00 (t\x94\x99"\x02q\x10y',
+    ],
```

Comment:
```suggestion
```

this is added purely for data gathering purposes, no need to fingerprint on it :slightly_smiling_face: